docs/cookbook/recipe_jquery_ui.rst

```diff
@@ -0,0 +1,39 @@
+Jquery UI
+=========
+
+The admin only comes with the `Jquery UI Sortable plugin`_.
+
+Adding another jQuery UI plugin
+-------------------------------
+
+To add another jQuery UI plugin in your admin, you have to tell
+webpack encore to not provide another instance of jQuery thanks
+to the `addExternals` method:
+
+.. code-block:: javascript
```

Comment:
maybe there is no lint configured for js?

Reply:
There is already https://github.com/sonata-project/SonataAdminBundle/blame/4.x/docs/reference/troubleshooting.rst#L47